src/main/java/frc/robot/Constants.java

```diff
@@ -180,4 +180,50 @@ public static final class Wrist {
         public static final double D = 0.0;
         public static final double F = 0.0;
     }
+
```

Comment:
Make all of these SCREAMIN_SNAKE format. As an action item, we should also make all the other constants SCREAMING_SNAKE case as well.